src/inc/msquic.h

```diff
@@ -875,6 +875,20 @@ typedef struct QUIC_STREAM_STATISTICS {
     uint64_t StreamBlockedByAppUs;
 } QUIC_STREAM_STATISTICS;
 
+typedef enum QUIC_AEAD_ALGORITHM_TYPE {
+    QUIC_AEAD_ALGORITHM_AES_128_GCM = 0,
+    QUIC_AEAD_ALGORITHM_AES_256_GCM = 1,
+    QUIC_AEAD_ALGORITHM_CHACHA20_POLY1305 = 2, // Not supported on Schannel
```

Comment:
Should we _fail_ then to set this type on Schannel?

Reply:
actually, in this case, we might be able to support this on Schannel; the only reason we don't support ChachaPoly with Schannel is because bcrypt doesn't allow us to set the counter value to a specific value, which is needed for the header protection.
But for the retry ticket, I don't think we're using a specific counter value, so it can start at 0 by default.